keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/AdminService.kt

```diff
@@ -63,7 +67,18 @@ class AdminService(
   }
 
   /**
-   * Updates artifact version records with the corresponding metadata, if available, by type [deb/docker/npm]
+   * For each artifact type [deb/docker/npm], run back-fill script to in case there's some missing data
+   */
+  @Scheduled(fixedDelayString = "\${keel.artifact-metadata-backfill.frequency:PT6H}")
+  fun updateBackFillArtifactMetadataForAllTypes(){
```

Comment:
Just a suggestion on naming:
```suggestion
  fun periodicallyBackFillArtifactMetadata(){
```